src/main/java/Task.java

```diff
@@ -0,0 +1,50 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     *
+     * @param description
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * gets status of task.
+     * 
+     * @return String
+     */
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    /**
+     * gets description of task
+     * 
+     * @return String
+     */
+    public String getDescription() {
+        return this.description;
+    }
+
+    /**
+     * sets work as done
+     */
+    public void taskDone() {
```

Comment:
Maybe this could be worded more as a verb to indicate what this method is doing precisely?
```suggestion
    public void markTaskAsDone() {
```

Reply:
will do so. Thank you for the help!